.github/workflows/docker.yml

```diff
@@ -0,0 +1,13 @@
+name: "Docker"
+
+on:
+  push:
+    branches: [ "main" ]
+    tags: [ "*" ]
+
+jobs:
+  build-fhiroxide:
```

Comment:
Name should also be updated here. Probably just `build` is fine, since the repo context is generally visible both in github UI and in emails?

Reply:
yep, done